tests/data_types.ts

```diff
@@ -20,11 +20,11 @@ testClient(async function inet() {
   const inet = "127.0.0.1";
   const insertRes = await CLIENT.query(
     "INSERT INTO data_types (inet_t) VALUES($1)",
-    inet,
+    { args: [inet] },
```

Comment:
What's the reason for this change?

Reply:
No worries, with the release of 1.0 right around the corner I completely understand it :) 
I might have misunderstood what you mean then. I the multiquery added support for splitting the queries if you just pass in a big query string, or if you pass in an array of strings. QueryType is also a part of the signature, so if you have muliple queries with independent arguments, that is also supported. This is currently the signature:

```
query(
    text: string,
    config?: QueryConfig,
  ): Promise<QueryResult>;

async multiQuery(
    queries: string | string[] | QueryType[],
  ): Promise<QueryResult[]>;

export type QueryType = {
  text: string;
  config?: QueryConfig;
};

export interface QueryConfig {
  args?: Array<unknown>;
  name?: string;
  encoder?: (arg: unknown) => EncodedArg;
}
```

The option would be to have this:
```
query(
    text: string,
    args?: Array<unknown>,
  ): Promise<QueryResult>;

async multiQuery(queries: QueryType[]): Promise<QueryResult[]>;

export type QueryType = {
  text: string;
  args?: Array<unknown>;
};
```

Is the latter maybe better to reduce the logic and/or possible bugs?

Reply:
The second one looks exactly like current `QueryConfig` interface :) so I guess no need to change the type?

Reply:
fair enough :)